### ISSUE TITLE
html-requires-loader.ts missing support of dep tracing on as-element with element and router-view

### ISSUE THREAD

**Issue by huochunpeng** [aurelia/webpack-plugin#141]
@jods4 missing support dep tracing of view and view-model.
```
<any as-element="compose" view="./v.html" view-model="./vm">
<any as-element="router-view" layout-view="./v.html" layout-view-model="./vm">
```

**Comment by fkleuver** [comment]
```
<any as-element="compose">
<any as-element="router-view">
```

These both work for me though, I just tested it with these versions:

```
"webpack": "^4.1.1"
"aurelia-webpack-plugin": "^3.0.0-rc.1"
```

What error(s) are you getting?

**Comment by huochunpeng** [comment]
I guess it's false alarm. I am just reading the code for other purpose, not use it.

**Comment by huochunpeng** [comment]
@fkleuver just want to make sure we are talking about the same thing.
I am talking about missing dependency tracing on "v" and "vm".

```
<any as-element="compose" view="./v.html" view-model="./vm">
```

If you only use `<any as-element="compose">` in local app, it doesn't matter. But this affects plugins use this kind of markup, webpack would not pack `./v.html` and `./vm` for files in node_modules.

**Comment by fkleuver** [comment]
Oh but when it comes to that sort of thing there's really no way around it except for maybe turning on the `includeAll` option. Webpack won't include anything that isn't referenced via `PLATFORM.moduleName` or some other explicit manner in the configuration.

**Comment by huochunpeng** [comment]
html-requires-loader.ts can trace that. This is an easy fix. I am just not motivated enough for a PR as I don't use webpack, so I reported it the lego master 😄 

I enjoy reading the code though. Very readable 👍

**Comment by fkleuver** [comment]
You're cracking me up man 😄

**Comment by huochunpeng** [comment]
There is deleting button on every post. If you want to delete yours, I will clear up mine. How is that sound?

**Comment by jods4** [comment]
Closed as a duplicate of #130
